commit ba4d4b71a35b933f407cbcd538abce2ad53559fb
Author:     nvkevlu <55759229+nvkevlu@users.noreply.github.com>
AuthorDate: Fri Jan 13 17:56:11 2023 -0500
Commit:     GitHub <noreply@github.com>
CommitDate: Fri Jan 13 14:56:11 2023 -0800

    add flare_api initial changes (#1161)
    
    * add flare_api changes, missed changes for training_cmds, add list_jobs add filter by user, add download_job_result implementation for redesigned AdminAPI
    
    * fix ci
    
    * add missed changes from training_cmds, add list_jobs
    
    * add filter on list_job by user, add download_job_result implementation for redesigned AdminAPI
    
    * add docs updates
    
    * fix bug for test with legacy string print for table, rename
    
    * fix ci
---
 nvflare/private/fed/client/client_engine.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/nvflare/private/fed/client/client_engine.py b/nvflare/private/fed/client/client_engine.py
index 2edc879e..61071b47 100644
--- a/nvflare/private/fed/client/client_engine.py
+++ b/nvflare/private/fed/client/client_engine.py
@@ -97,6 +97,7 @@ class ClientEngine(ClientEngineInternalSpec):
         running_jobs = []
         for job_id in self.get_all_job_ids():
             run_folder = os.path.join(self.args.workspace, WorkspaceConstants.WORKSPACE_PREFIX + str(job_id))
+            app_name = ""
             app_file = os.path.join(run_folder, "fl_app.txt")
             if os.path.exists(app_file):
                 with open(app_file, "r") as f:
